### ISSUE TITLE
Use contrast ratio while calculating text color instead of lightness

### ISSUE THREAD

**Issue by m0ksem** [epicmaxco/vuestic-ui#3337]
Now we use lightness and check if it is > 200 then we use black or white. How about to use contrast and follow WCAG recommendations?
